java/Cherepnin_Oleh/src/com/zoolatech/lecture6/tasks/_1/City.java

```diff
@@ -0,0 +1,49 @@
+package com.zoolatech.lecture6.tasks._1;
+
+import java.util.Objects;
+
+public class City {
+    private String name;
+    private String country;
+    private int population;
+
+    public City(String name, String country, int population) {
+        this.name = name;
+        this.country = country;
+        this.population = population;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (!(o instanceof City)) return false;
+        City city = (City) o;
+        return getPopulation() == city.getPopulation() && getName().equals(city.getName()) && getCountry().equals(city.getCountry());
```

Comment:
inside of a class you can access instance variables directly, without getter methods